AllReadyApp/Web-App/AllReady/config.json

```diff
@@ -18,13 +20,14 @@
     "DefaultConnection": {
       "ConnectionString": "Server=(localdb)\\MSSQLLocalDB;Database=AllReady;Integrated Security=true;MultipleActiveResultsets=true;"
     },
-    "InsertSampleData": "true",
-    "InsertTestUsers": "true",
     "Storage": {
       "AzureStorage": "[storagekey]",    
       "EnableAzureQueueService": "false"
     }
   },
+  "Email": {
+    "EmailFolder": "..."
+  },
```

Comment:
This setting is used in the email sender but didn't have an entry in the config.